drivers/usb_c/tcpc/ucpd_stm32.c

```diff
@@ -1022,43 +1081,42 @@ static void ucpd_isr(const struct device *dev_inst[])
 	 * Multiple UCPD ports are available
 	 */
 
-	uint32_t ucpd_base;
+	uint32_t sr1;
+	uint32_t sr2;
 
 	/*
 	 * Since the UCPD peripherals share the same interrupt line, determine
 	 * which one generated the interrupt.
 	 */
-	if (LL_SYSCFG_IsActiveFlag_UCPD1()) {
+
+	/* Read UCPD1 Status Register */
```

Comment:
This comment is misleading.
You have no guarantee `dev_inst[0]` is `UCPD1`

Reply:
Fix in following change list